lms/djangoapps/discussion_api/tests/test_forms.py

```diff
@@ -142,6 +143,10 @@ def test_mutually_exclusive(self, params):
             "The following query parameters are mutually exclusive: topic_id, text_search, following"
         )
 
+    def test_invalid_view_choice(self):
+        self.form_data["view"] = "not_a_valid_choice"
+        self.assert_error("view", "Select a valid choice. not_a_valid_choice is not one of the available choices.")
```

Comment:
Agreed. In my experience, it's best to use string literals in tests, because if you refer to a constant or function in the implementation, the test will still pass even if you change the constant or function to nonsense.